### PR TITLE
Update Plex Rockon Documentation

### LINKED ISSUE #214: [Rock-ons] Update Plex write-up with hardware transcoding
Since the [update of our Plex rock-on to support hardware transcoding using QuickSync](https://github.com/rockstor/rockon-registry/pull/204), it is proposed to add detailed instructions on how to set it up to the existing write-up:
http://rockstor.com/docs/docker-based-rock-ons/plex-media-server.html

Instructions could cover:
- how to verify hardware compatibility: https://ark.intel.com/content/www/us/en/ark/search/featurefilter.html?productType=873&0_QuickSyncVideo=True
- how to get the path needed during the rock-on install (most likely `/dev/dri` for QuickSync)
- share to add (`/transcode`)
- how to enable and set it from within the Plex-UI: https://support.plex.tv/articles/115002178853-using-hardware-accelerated-streaming/

### PR DESCRIPTION
**Pull request by Hooverdan96**
Fixes #214 and #172 .

Update of Plex Rockon Documentation:
- newer screenshots
- refactoring descriptions
- include transcoding explanation
- add workaround when no Plex account is used

- [X] With the proposed changes no Sphinx errors or warnings are generated.
- [X] I have added my name to the AUTHORS file, if required (descending alphabetical order).

### CODE REVIEW

**Comment by Hooverdan96** [comment]
These are the three linkcheck failures:
```
(interface/docker-based-rock-ons/plex-media-server: line   54) broken    https://ark.intel.com/ - 403 Client Error: Forbidden for url: https://ark.intel.com/
(interface/system/update_channels: line   16) broken    https://opensource.org/license/mit-0 - 403 Client Error: Forbidden for url: https://opensource.org/license/mit-0
(interface/system/update_channels: line   16) broken    https://opensource.org/license/apache-2-0 - 403 Client Error: Forbidden for url: https://opensource.org/license/apache-2-0
```
However, when manually accessing these three, they come up fine (no visible redirects or other address changes).

**Comment by Hooverdan96** [comment]
@FroggyFlox, @phillxnet I'm sure I missed some stuff, please take a look and make suggestions for change.

**Comment by phillxnet** [comment]
@Hooverdan96 Just having a looks at this now:
We seem to have gained the following:

```
(interface/docker-based-rock-ons/plex-media-server: line   54) broken    https://ark.intel.com/ - 403 Client Error: Forbidden for url: https://ark.intel.com/
```
But works fine in normal browser and redirects to: https://ark.intel.com/content/www/us/en/ark.html

So this may be a candidate, when re-visiting, to add our ignored list as we have the two you posted here already to add also.

Just reviewing changes currently, and this is not a show-stopper so no worries really. Just wanted to add to the known flaky urls as and when we see them.

**Comment by Hooverdan96** [comment]
The admin user was not planned. I just had it created in that "demo" instance that way. I can clarify (or create a different screenshot for that.
and the formatting above, was not intentional. I think that's how it was before, and I just moved the text block. Let me fix that.

**Comment by Hooverdan96** [comment]
@phillxnet, I rebased the fork based on your most recent merge, updated the User screenshot and fixed the link you mentioned above.

File: images/interface/docker-based-rock-ons/plex_user_info.png
Comment (phillxnet) [inline images/interface/docker-based-rock-ons/plex_user_info.png:1]: @Hooverdan96 That is better. My apologies for taking so long with the review. It looks like you have fixed this while I was still reviewing. This way the picture tallies with a non-admin user as per the text. And avoids folks creating users as Web-UI admin as they think it's necessary for Rock-on function or what-ever.

File: interface/docker-based-rock-ons/plex-media-server.rst [hunk 1]
```diff
@@ -137,208 +148,251 @@ This is the **Default Port** and it is unlikely that you will have to alter it.
...
 
-.. image:: /images/interface/docker-based-rock-ons/plex_tos.png
-   :width: 100%
+.. image:: /images/interface/docker-based-rock-ons/plex_login.png
+   :width: 90%
    :align: center
 
-It is required that you *AGREE* in order to proceed with the server setup.
+.. note::
+    Please see `Sign in to Your Plex Account<https://support.plex.tv/articles/200878643-sign-in-to-your-plex-account/>`
```
[the remark this answers is not in the document]
Comment (Hooverdan96) [reply interface/docker-based-rock-ons/plex-media-server.rst]: already done, while you were reviewing

File: images/interface/docker-based-rock-ons/plex_share_owner.png
[the remark this answers is not in the document]
Comment (Hooverdan96) [reply images/interface/docker-based-rock-ons/plex_share_owner.png:1]: addressed while you were reviewing. New screenshot inserted that doesn't show `plex` as an admin user.

**Comment by Hooverdan96** [comment]
@phillxnet, ok I think I have now captured everything I can think of, including all of your suggestions.

**Comment by phillxnet** [comment]
@Hooverdan96 The only thing I see amiss after these changes is that the plex_share_owner.png file still looks to be root.root with default settings. Should this not be plex.plex with no 'other' access? With this share config Plex would have no write access. As I suspect you have the image there, do you fancy popping that in on this PR. We can do anther time: I'm just not keen of publishing as-is given the no write access and non optimal owner.group. Given the text is all nice now, and we have the new non-admin plex user image.

**Comment by Hooverdan96** [comment]
no, you're right, I thought I had done it ... starting too long at your own writing/pictures makes you overlook crucial things. I will update the view.

**Comment by Hooverdan96** [comment]
@phillxnet, ok I misread one of your earlier suggestions and only associated that with the plex user being part of the admin group. The share owner under the Access Control tab screenshot is now updated as well.

**Review by phillxnet — APPROVED** [review]

**Comment by phillxnet** [comment]
@Hooverdan96 Thanks again for all this effort, this is a major improvement and it's nice to now have this key guide updated re the transcoding option/Share. And to have more refined explanations etc.

About to merge and get this published shortly.

**Comment by phillxnet** [comment]
PR product PRODUCTION published.

**Pull request merged**